src/pages/admin/AdminSessionPage.tsx

```diff
@@ -0,0 +1,217 @@
+import { useEffect, useState, FormEvent } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import {
+  Box,
+  Button,
+  VStack,
+  HStack,
+  Input,
+  Text,
+  Card,
+  CardBody,
+  CardHeader,
+  Heading,
+  FormControl,
+  FormLabel,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { ArrowBackIcon } from '@chakra-ui/icons';
+import { Calendar, Clock, Key, Users } from 'lucide-react';
+import { Link } from 'react-router-dom';
+import api from '../../services/api';
+
+interface AttendanceSession {
+  id: string;
+  title: string;
+  key: string;
+  expires: string;
+  attendees: string[];
+}
+
+const isSessionActive = (session: AttendanceSession): boolean => {
+  return new Date(session.expires) > new Date();
+};
+
+export default function AdminSessionPage() {
+  const [sessions, setSessions] = useState<AttendanceSession[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [title, setTitle] = useState('');
+  const [key, setKey] = useState('');
+  const [expiresDate, setExpiresDate] = useState<Date | null>(null);
```

Comment:
instead of `<Date | null>(null)`, just use `<Date>()`. This will give type `Date | undefined` and is a little cleaner

Reply:
the date picker onChange has `onChange?: (date: Date[] | null,...`